todo_app/services/trello.py

```diff
@@ -0,0 +1,91 @@
+import requests
+
+from flask import current_app as app
+
+from todoItem import ToDoItem
+
+board_url = "https://api.trello.com/1/boards/{id}"
+
+headers = {
+   "Accept": "application/json"
+}
+
+def get_trello_creds():
+    return { 'key': app.config['TRELLO_KEY'], 'token': app.config['TRELLO_TOKEN'] }
+
+def get_list_params():
+    return { 'key': app.config['TRELLO_KEY'], 'token': app.config['TRELLO_TOKEN'], 'cards': 'open' }
+
+def add_card_params(title, item_list):
+    return { 'key': app.config['TRELLO_KEY'], 'token': app.config['TRELLO_TOKEN'], 'name': title, 'idList': item_list['id'] }
+
+def update_card_params(item_list):
+    return { 'key': app.config['TRELLO_KEY'], 'token': app.config['TRELLO_TOKEN'], 'idList': item_list['id'] }
+
+def get_url(endpoint):
+    return "https://api.trello.com/1/" + endpoint
+
+def get_boards():
+    
+    params = get_trello_creds()
+    url = get_url('members/me/boards')
+
+    response = requests.get(url, params)
+    
+    return response.json()
+
+def get_lists():
+
+    get_boards()
+    
+    params = get_list_params()
+    url = get_url('boards/%s/lists' % '6006195ae74f187322fda449')
```

Comment:
Rather than hardcoding the board id, we should get it from an environment variable - otherwise your code will only run for a single trello board & can't be configured for different environments